packages/contracts/contracts/optimistic-ethereum/libraries/wrappers/Lib_ExecutionManagerWrapper.sol

```diff
@@ -118,6 +118,25 @@ library Lib_ExecutionManagerWrapper {
         return abi.decode(returndata, (address));
     }
 
+    /**
```

Comment:
This wrapper function is redundant with ability to use `assembly { id := chainid() }` as previously discussed.  If we think that's ugly we can keep it but guessing this was vestigial based on that feedback getting incorporated previously

Reply:
Yeah, I think the assembly is a bit ugly. Wrapper seemed nicer to me.